### PR TITLE
Remove redundant Error messages

### PR DESCRIPTION
**Pull request by taysea**
#### What does this PR do?

Removing redundant uses of custom `Error` component that are now being handled by the theme. I have also removed the local theme changes now that they have merged into grommet-theme-hpe.

#### Where should the reviewer start?
Any file.

#### What testing has been done on this PR?
Tested locally.

#### How should this be manually tested?
Check deploy on changed files. Only one `CircleAlert` icon should appear on the error state.

#### Should this PR be mentioned in Design System updates?
No.

#### Is this change backwards compatible or is it a breaking change?
Yes, backwards compatible.

### CODE REVIEW

File: aries-site/src/themes/aries.js [hunk 1]
```diff
@@ -1,23 +1,12 @@
 import { hpe } from 'grommet-theme-hpe';
 import { deepMerge } from 'grommet/utils';
-import { CircleAlert, Unsorted } from 'grommet-icons';
```
Comment (halocline) [inline aries-site/src/themes/aries.js:3]: 👍

**Review by halocline — COMMENTED** [review]
One more: https://github.com/grommet/hpe-design-system/blob/afe14ef6ca23662d7bf7d975221bb49396561463/aries-site/src/examples/templates/wizard/WizardValidationExample.js#L377

Can we remover this one as well?

**Comment by taysea** [comment]
> One more:
> 
> https://github.com/grommet/hpe-design-system/blob/afe14ef6ca23662d7bf7d975221bb49396561463/aries-site/src/examples/templates/wizard/WizardValidationExample.js#L377
> 
> Can we remover this one as well?

I believe this one is a custom step level error message, in which case we still need it because it's not supported out of the box by grommet. It appears at the bottom of a step if there is an issue with an input.

**Comment by halocline** [comment]
> > One more:
> > https://github.com/grommet/hpe-design-system/blob/afe14ef6ca23662d7bf7d975221bb49396561463/aries-site/src/examples/templates/wizard/WizardValidationExample.js#L377
> > 
> > Can we remover this one as well?
> 
> I believe this one is a custom step level error message, in which case we still need it because it's not supported out of the box by grommet. It appears at the bottom of a step if there is an issue with an input.

Ahh, gotcha. That makes sense.

**Review by halocline — APPROVED** [review]

**Pull request merged**